tools/publishIsolate.ts

```diff
@@ -5,7 +5,7 @@ import getNpmPackageVersion from 'get-npm-package-version';
 import { compareVersions } from 'compare-versions';
 
 async function main() {
-  const isolatePackageRoot = './dist/isolate';
+  const isolatePackageRoot = './dist/@js-sdsl/isolate';
```

Comment:
That was not what I meant. xD

```typescript
const packagePath = path.join(isolatePackageRoot, packageName);
console.log(packagePath);  // output 'dist/isolate/@js-sdsl/xxx' expected 'dist/isolate/xxx'
```

It should replace '@js-sdsl' to ''.

Reply:
Now I understand